src/gnark_backend_wrapper/groth16/acir_to_r1cs.rs

```diff
@@ -39,7 +38,10 @@ impl RawR1CS {
             .opcodes
             .into_iter()
             .filter(acvm::Opcode::is_arithmetic)
-            .map(|opcode| RawGate::new(opcode.arithmetic().unwrap()))
+            .map(|opcode| match opcode.arithmetic() {
+                Some(expression) => RawGate::new(expression),
+                None => RawGate::new(acvm::Expression::default()),
+            })
```

Comment:
We'll allow this for this PR, but we should add a TODO (and add a ticket in Trello) explaining that we should handle the error instead of using `Expression::default`.